### DISCUSSION TITLE
Best Model for Removing Screamo / Metal Screams or growls vocals?

### DISCUSSION THREAD

**Discussion by radwalls** [Anjok07/ultimatevocalremovergui#753] [General]
Love this application, but on some metal songs the screams remain. Been using the Karaoke model

**Comment by NeonPro66** [comment]
Same here. What are your best settings for metal?

**Comment by radwalls** [comment]
Try a high intensity it managed to get everythingOn Aug 28, 2023, at 11:52 PM, NeonPro66 ***@***.***> wrote:﻿
Same here. What are your best settings for metal?
